static/src/javascripts/lib/cookies.spec.js

```diff
@@ -6,6 +6,7 @@ import {
     addForMinutes,
     removeCookie,
     getCookie,
+    isValidCookieValue,
```

Comment:
Are you exporting `isValidCookieValue` purely for testing purposes? If so you could follow the pattern used elsewhere to export `export const _ = { isValidCookieValue };` instead?